Northstar.CustomServers/mod/scripts/vscripts/_items.nut

```diff
@@ -5700,6 +5700,12 @@ bool function IsSubItemLocked( entity player, string ref, string parentRef )
 {
 	if ( DevEverythingUnlocked( player ) )
 		return false
+	
+	if ( IsItemPurchasableEntitlement( ref, parentRef ) )
+		return false
+	
+	if ( GetItemType( ref ) == eItemTypes.PRIME_TITAN || GetSubitemType( parentRef, ref ) == eItemTypes.PRIME_TITAN )
```

Comment:
Why do you need to check either case instead of just the one? Is it to handle prime executions or something?

Reply:
Yeah prime executions checks if player have Primes unlocked, otherwise they can't be used.